### ISSUE TITLE
AttributeError as modules can be string as well as dict

### ISSUE THREAD

**Issue by ahayzen** [flathub-infra/flatpak-external-data-checker#2]
When parsing the Skype json the following error occurs.
```
./flatpak-external-data-checker ~/Downloads/com.skype.Client.json --filter-type extra-data
Traceback (most recent call last):
  File "./flatpak-external-data-checker", line 70, in <module>
    manifest_checker = checker.ManifestChecker(args.manifest)
  File "/home/andrew/Workspace/Projects/flatpak/external-data-checker/jrocha/flatpak-external-data-checker/src/checker.py", line 45, in __init__
    self._collect_external_data()
  File "/home/andrew/Workspace/Projects/flatpak/external-data-checker/jrocha/flatpak-external-data-checker/src/checker.py", line 48, in _collect_external_data
    self._external_data = self._get_module_data_from_json(self._json_data) + \
  File "/home/andrew/Workspace/Projects/flatpak/external-data-checker/jrocha/flatpak-external-data-checker/src/checker.py", line 71, in _get_module_data_from_json
    for source in module.get('sources', []):
AttributeError: 'str' object has no attribute 'get'
```

This is due to modules also being str's and not dicts https://github.com/flathub/com.skype.Client/blob/master/com.skype.Client.json#L66

I have a simple patch incoming.

**Comment by joaquimrocha** [comment]
(Sorry for the delay)
This makes sense, but do you mind adding a test for this? (should be simple too)

**Comment by wjt** [comment]
I fixed this in 6b1c30f2e70e778d17e508ab1e6dc2271747a4de

**Comment by joaquimrocha** [comment]
Right, I forgot to close this.
